src/Weather.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from "react";
+import "./App.css";
+
+class Weather extends Component {
+  state = {
+    temp: "",
+    pic: "",
+    description: "",
+  };
+
+  componentDidMount() {
```

Comment:
Another great opportunity for async/await, introduced in ES8. They are really nice at preventing these chains of .then statements.